GetIntrinsic.js

```diff
@@ -177,19 +183,30 @@ var stringToPath = function stringToPath(string) {
 };
 /* end adaptation */
 
-var getBaseIntrinsic = function getBaseIntrinsic(name, allowMissing) {
-	if (!(name in INTRINSICS)) {
-		throw new SyntaxError('intrinsic ' + name + ' does not exist!');
-	}
+var getBaseIntrinsic = function getBaseIntrinsic(baseNames, allowMissing) {
+	var intrinsicName;
+
+	for (var i = baseNames.length - 1; i >= 0; i--) {
+		var name = baseNames[i];
+		intrinsicName = '%' + name + '%';
+		if (hasOwn(INTRINSICS, intrinsicName)) {
+			var value = INTRINSICS[intrinsicName];
+			if (typeof value === 'undefined' && !allowMissing) {
+				throw new $TypeError('intrinsic ' + intrinsicName + ' exists, but is not available. Please file an issue!');
+			}
 
-	// istanbul ignore if // hopefully this is impossible to test :-)
-	if (typeof INTRINSICS[name] === 'undefined' && !allowMissing) {
-		throw new $TypeError('intrinsic ' + name + ' exists, but is not available. Please file an issue!');
+			return {
+				name: name,
+				nextIndex: i + 1,
+				value: value
+			};
+		}
 	}
 
-	return INTRINSICS[name];
+	throw new SyntaxError('intrinsic ' + intrinsicName + ' does not exist!');
 };
 
+// eslint-disable-next-line max-lines-per-function, max-statements
```

Comment:
just bump the value in eslintrc rather than disabling it inline

Reply:
Doing that causes **eslint** to complain that many of the `eslint‑disable‑next‑line` directives in the `201x` directories have no effect.

Reply:
ah, ok then this is fine :-)